lib/pages/job_settings_page.rb

```diff
@@ -47,5 +56,53 @@ def configure_rake_task(target,working_directory)
       task_target.set(target)
       task_working_directory.set(working_directory)
     end
+
+    def configure_more_task(command,arglist,working_directory,run_conditions)
+      task_commands.set(command)
+      arglist.split(',').each {|arg| argument.set(arg)}
+      task_working_directory.set(working_directory)
+
+      run_conditions.split(',').each { |run_condition|
+         page.find("input[name='task[runIfConfigs#{run_condition}]']").set(true)
+        }
+    end
+
+    def  configure_on_cancel_more_task(task,command)
+      on_cancel_task_check_box.click
+      page.find('option', text: task).click
+      task_advance_option_command.set(command)
+    end
+  
+    def get_cell_value_from_table(row_count,header_name,column_name)
+       case column_name
+         when "Task Type"
+          return page.find(".tasks_list_table tr:nth-child(#{row_count.to_i}) td:nth-child(2)").text.eql?(header_name)
+         when "Run If Conditions"
+          return page.find(".tasks_list_table tr:nth-child(#{row_count.to_i}) td:nth-child(3)").text.eql?(header_name)
+         when "Properties"
+          return page.find(".tasks_list_table tr:nth-child(#{row_count.to_i}) td:nth-child(4)").text.gsub("\n", ' ').eql?(header_name)
+         when "On Cancel"
+          return page.find(".tasks_list_table tr:nth-child(#{row_count.to_i}) td:nth-child(5)").text.eql?(header_name)
+        end
+     end   
+  
+   def delete_task (task_no)
+    page.find("tr:nth-child(#{task_no.to_i}) td span.icon_remove").click
+    delete_task_button.click
+   end 
+
+   def verify_task_with_command_is_not_exists(command)
+     page.has_no_css?('li.task_property.command span.value', text:command.split(':').last.strip)
+   end
+
+   def verify_task_with_command_is_not_exist_in_config(task,command,job,stage,pipeline)
+    
+      if basic_configuration.get_config_from_server.xpath("//cruise/pipelines/pipeline[@name='#{scenario_state.actual_pipeline_name(pipeline)}']/stage[@name='#{stage}']/jobs/job[@name='#{job}']/tasks/#{task}[@command='#{command}']").count==0 
+        return true 
```

Comment:
There is no need for this if statement here, it can be just the condition, whatever the condition statment evaluates to will be returned either true or false. So this can be without the if statement